Doppler.BigQueryMicroservice/Controllers/BigQueryController.cs

```diff
@@ -41,14 +58,38 @@ public async Task<ActionResult> GetAllowedEmails(string accountName)
         [HttpPut("/big-query/{accountName}/allowed-emails")]
         public async Task<ActionResult> SaveAllowedEmails(string accountName, AllowedEmails model)
         {
-            var user = await _userRepository.GetUserByEmail(accountName);
-            if (user == null)
+            try
             {
-                return NotFound();
+                bool notified = false;
+                var user = await _userRepository.GetUserByEmail(accountName);
+                if (user == null)
+                {
+                    return NotFound();
+                }
+
+                var results = await _userAccessByUserRepository.MergeEmailsAsync(user.IdUser, model.Emails);
+                var newEmails = results.Where(a => a.Action == "INSERT").Select(e => e.Email).ToList();
+                var template = _emailSettings.Value.BigQueryInvitationTemplateId[model.Languaje.ToLower()];
+                if (newEmails.Count > 0)
+                {
+                    try
+                    {
+                        notified = await _emailSender.SendWithTemplateAsync(template, newEmails, model.Emails);
+                    }
+                    catch (System.Exception ex)
+                    {
+                        _logger.LogError(ex, "Error in method SendWithTemplateAsync, service {service}", nameof(EmailSender));
```

Comment:
[The same here](https://github.com/FromDoppler/big-query-microservice/pull/7/files#r719339203), it is really minor, but I would prefer a human-friendly message and avoid redundant information, something like:

```suggestion
                        _logger.LogError(ex, "Error notifying new allowed emails");
```